app/models/trade.rb

```diff
@@ -0,0 +1,12 @@
+class Trade < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :categories
+
+  validates :name, presence: true
+
+  validates :name, length: { maximum: 50 }
+
+  validates :amount, presence: true
+
```

Comment:
- [ ] I think there should be a validation for catgegory as per project requirements. I am able to add a category without the category field, but the project requirement says it should be mandatory.
![image](https://github.com/obed300/budget-app/assets/62849872/d59eec7e-d3f2-4272-9099-4c6405c28385)